### DISCUSSION TITLE
Running example code results in an error message[AttributeError: 'Categorical' object has no attribute 'dist']

### DISCUSSION THREAD

**Discussion by LiuXing1122** [RLE-Foundation/rllte#42] [Q&A] [answered]
When I run your example code, the result is an error as follows:

  File "/*******/rllte/xploit/policy/on_policy_shared_actor_critic.py", line 147, in forward
    def forward(self, obs: th.Tensor, training: bool = True) -> Tuple[th.Tensor, Dict[str, th.Tensor]]:
  File "/*******/rllte/xplore/distribution/categorical.py", line 99, in mean
    return self.dist.probs.argmax(axis=-1)
AttributeError: 'Categorical' object has no attribute 'dist'

I don't know why this error is reported, may I ask if my library version is incorrect?

**Comment by friedrichyuan** [comment, marked as answer]
Sorry for the bug. Can you tell me which example did you run?

**Reply by LiuXing1122** [reply]
I'm sorry, I did report this error earlier, but later on, I don't know why it didn't go wrong. Maybe there was an issue with the PyCharm I used. Thank you for your reply.
